src/store/createStore.ts

```diff
@@ -43,6 +81,36 @@ export const createStore = (options: CreateStoreOptions) =>
               state.account.publicKey = "";
             }),
         },
+        // Explore Endpoints
+        exploreEndpoints: {
+          ...initExploreEndpointState,
+          updateNetwork: (network: Network) =>
```

Comment:
I can't tell from the design. Can `exploreEndpoints.network` be different from the top level `network`?

Reply:
So, I made it this way considering that we can save endpoints in the future. And we'll need to include the network with that endpoint to make sure it functions properly.

It also helps to clear the current endpoint data in the store if the top level network changes.